mappings/net/minecraft/item/CompassItem.mapping

```diff
@@ -15,3 +15,14 @@ CLASS net/minecraft/class_1759 net/minecraft/item/CompassItem
 		ARG 0 tag
 	METHOD method_26365 hasLodestone (Lnet/minecraft/class_1799;)Z
 		ARG 0 stack
+	CLASS 1
+		FIELD field_23978 targeted Lnet/minecraft/class_1759$class_5171;
+		FIELD field_23979 targetless Lnet/minecraft/class_1759$class_5171;
+	CLASS class_5171 AngleRandomizer
+		FIELD field_23980 value D
+		FIELD field_23981 speed D
+		FIELD field_23982 lastUpdateTime J
+		METHOD method_27188 get ()D
```

Comment:
`targeted.get()` looks weird. Shouldn't be either `targetedAngleRandomizer.get()` or `targeted.getAngle()`?

Reply:
`getAngle` sounds good.